etna/ciim/models.py

```diff
@@ -182,7 +182,7 @@ def __init__(self, model):
         self.model = model
 
     def get(self, iaid=None, reference_number=None, expand=False):
-        response = self.client.fetch(iaid=iaid, reference_number=reference_number, expand=expand)
+        response = self.client.fetch(iaid=iaid, id=reference_number, expand=expand)
 
```

Comment:
@danbentley - Currently we are 'fecth'ing with id=None. `{record_detail_view->Record.search.get(iaid, expand=True), SearchManager.get()} ` since 'id' is Generic identifier to fetch, Thought we can we have 'reference_number' replaced with 'id' to match kong api param used for fetch.

Reply:
Thanks Allan, I agree.

It's not something I can address in this PR but it will be addressed in subsequent work.